package.json

```
@@ -43,7 +43,7 @@
     "abbrev": "^2.0.0",
     "archy": "^1.0.0",
     "bluebird": "^3.7.2",
-    "hexo-cli": "^4.3.2",
+    "hexo-cli": "^4.3.0",
```

```suggestion
    "hexo-cli": "^4.3.2",
```
No need to revert this one.